src/SignalRClient/BenchmarksEventSource.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Diagnostics.Tracing;
+
+namespace SignalRClient
+{
+    internal sealed class BenchmarksEventSource : EventSource
```

Comment:
@sebastienros does everyone end up copying this around?

Reply:
Just Brennan, initiation ritual. Will put it in a class library at least.